content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1040,16 +1096,41 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `MixedProtocolLBService`：允许在同一 `LoadBalancer` 类型的 Service 实例中使用不同的协议。
 <!--
 - `MultiCIDRRangeAllocator`: Enables the MultiCIDR range allocator.
-- `NetworkPolicyEndPort`: Enable use of the field `endPort` in NetworkPolicy objects,
-  allowing the selection of a port range instead of a single port.
+- `MultiCIDRServiceAllocator`: Track IP address allocations for Service cluster IPs using IPAddress objects.
 - `NetworkPolicyStatus`: Enable the `status` subresource for NetworkPolicy objects.
+- `NewVolumeManagerReconstruction`: Enable improved discovery of mounted volumes during kubelet
```

Comment:
Duplicated `NewVolumeManagerReconstruction`.

Is it a mistake? @tengqm

Reply:
Duplication is an error for sure.